lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -824,6 +836,49 @@ getConfigParam = label "getConfigParam" $ do
     secondsToNominalDiffTime =
         toEnum . fromEnum . secondsToDiffTime .  fromIntegral
 
+    getRewardFormula :: Get RewardFormula
+    getRewardFormula = do
+        tag <- getWord8
+        case tag of
+            1 -> LinearFormula <$> getRewardParams
+            2 -> HalvingFormula <$> getRewardParams
+            _ -> fail $ "Unknown reward parameters with tag " ++ show tag
+
+    -- NOTE: getHalvingParams and getLinearParams are currently identical.
+    --
+    -- https://github.com/input-output-hk/chain-libs/blob/149e01cf61a9eef2660a6944fe0acc73d490e70f/chain-impl-mockchain/src/config.rs#L404-L442
+    getRewardParams :: Get RewardParams
+    getRewardParams = do
+        start <- getWord64be
+        ratio <- getRatio
+        epochStart <- getWord32be
+        epochRate <- getWord32be
+        return $ RewardParams
+                { rStart = start
+                , rRatio = ratio
+                , rEpochStart = epochStart
+                , rEpochRate = epochRate
+                }
+
+    getRatio :: Get Ratio
+    getRatio = do
+        ratioNum <- getWord64be
+        ratioDen <- getWord64be
+        return $ Ratio ratioNum ratioDen
+
+
+    getRewardLimitByAbsoluteStake :: Get Ratio
+    getRewardLimitByAbsoluteStake = getRatio
```

Comment:
This function seems superfluous and adds an unnecessary indirection. I'd suggest to simply use `getRatio`.